dali/operators/reader/loader/video/video_loader_decoder_cpu.cc

```diff
@@ -18,6 +18,7 @@ namespace dali {
 void VideoLoaderDecoderCpu::PrepareEmpty(VideoSample<CPUBackend> &sample) {
   sample = {};
   sample.data_.set_pinned(false);
+  sample.data_.SetLayout("FHWC");
```

Comment:
Another small bug fix I found during the tests rework - layout was not set in CPU reader.